docs/docs/en/index.md

```diff
@@ -1,24 +1,67 @@
 ---
 title: What is Tuist?
 description: Extend your Apple native tooling to better apps at scale.
+layout: home
+hero:
+  name: Tuist
+  text: From idea to the store
+  tagline: Extend Apple's toolchain to build better apps faster
+  image:
+    src: /logo.png
+    alt: Tuist
+  actions:
+    - theme: brand
+      text: What is Tuist?
+      link: /en/guides/tuist/about
+    - theme: alt
+      text: Quick start
+      link: /en/guides/quick-start/install-tuist
+    - theme: alt
+      text: View on GitHub
+      link: https://github.com/tuist/tuist
+features:
+  - icon: 📝
+    title: Projects
+    details: A Swift-based DSL to make Xcode projects more managleable and scalable.
+    linkText: "Create or migrate project"
+    link: "/en/guides/develop/projects"
+  - icon: 📦
+    title: Cache
+    details: |
+      <div style="margin-bottom: 1rem; background: var(--vp-custom-block-tip-code-bg); color: var(--vp-c-tip-1); font-size: 11px; display: inline-block; padding-left: 5px; padding-right: 5px; border-radius: 10%;">Requires a Tuist project</div>
```

Comment:
Should we change references of `Tuist project` to `Generated project`, in this case to `Requires a generated project` as part of our effort to make clearer the distinction between "local" and "remote" Tuist projects?